### PR TITLE
Added SymPy into the org list and the ideas page.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

File: organization-team.md [hunk 1]
```diff
@@ -10,7 +10,24 @@ organizing Google Summer of Code 2015 for NumFOCUS.
...
+
 - andy.terrel
 - gvwilson
 - mjsottile
 - wking
+
+### SymPy
+
+- Aaron Meurer (asmeurer)
+- Mateusz Paprocki (mattpap)
```
Comment (r-gaia-cs) [inline organization-team.md]: mattpag isn't a valid username to Melange.

File: organization-team.md [hunk 2]
```diff
@@ -10,7 +10,24 @@ organizing Google Summer of Code 2015 for NumFOCUS.
...
+- Jim Crist (jcrist)
```
Comment (r-gaia-cs) [inline organization-team.md]: jcrist isn't a valid username to Melange.

File: organization-team.md [hunk 3]
```diff
@@ -10,7 +10,24 @@ organizing Google Summer of Code 2015 for NumFOCUS.
...
+- Ondřej Čertík (certik)
+- Tim Lahey (tjlahey)
```
Comment (r-gaia-cs) [inline organization-team.md]: tjlahey isn't a valid username to Melange.

File: organization-team.md [hunk 4]
```diff
@@ -10,7 +10,24 @@ organizing Google Summer of Code 2015 for NumFOCUS.
...
+- Thilina Rathnayake (thilinarmtb)
+- Harsh Gupta (hargup)
```
Comment (r-gaia-cs) [inline organization-team.md]: hargup isn't a valid username to Melange.

File: organization-team.md [hunk 5]
```diff
@@ -10,7 +10,24 @@ organizing Google Summer of Code 2015 for NumFOCUS.
...
+- David Li (lidavidm96)
```
Comment (r-gaia-cs) [inline organization-team.md]: lidavidm96 isn't a valid username to Melange.

File: organization-team.md [hunk 6]
```diff
@@ -10,7 +10,24 @@ organizing Google Summer of Code 2015 for NumFOCUS.
...
+- Sean Vig (flacjacket)
+- Jason Moore (moorepants)
+- Gilbert Gede (gilbertgede)
```
Comment (r-gaia-cs) [inline organization-team.md]: gilbertgede isn't a valid username to Melange.

File: organization-team.md [hunk 7]
```diff
@@ -10,7 +10,24 @@ organizing Google Summer of Code 2015 for NumFOCUS.
...
+- Sachin Joglekar (srjoglekar246)
```
Comment (r-gaia-cs) [inline organization-team.md]: srjoglekar246 isn't a valid username to Melange.

File: organization-team.md [hunk 8]
```diff
@@ -10,7 +10,24 @@ organizing Google Summer of Code 2015 for NumFOCUS.
 
 ## Mentor
 
+# Software Carpentry
```
Comment (r-gaia-cs) [inline organization-team.md]: Should start with `###`.

**Comment by r-gaia-cs** [comment]
@moorepants I'm happy that Sympy will be with us. I couldn't add some mentors under NumFOCUS umbrella. Could you help me?

**Comment by moorepants** [comment]
All of our mentors haven't necessarily made their Melange profiles for this year. The usernames in this list are github user names. We are working on getting everyone to create new melange profiles. I'll let you know when that is done.

**Comment by moorepants** [comment]
@certik @asmeurer We need to get all our mentors to create new melange profiles.

**Comment by r-gaia-cs** [comment]
@moorepants I merge this locally, please review it.

**Pull request merged**